ci-operator/step-registry/ipi/conf/aws/sharednetwork/ipi-conf-aws-sharednetwork-commands.sh

```diff
@@ -10,7 +10,7 @@ curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -
 CONFIG="${SHARED_DIR}/install-config.yaml"
 PATCH="${SHARED_DIR}/install-config-sharednetwork.yaml.patch"
 
-aws_region=$(/tmp/yq r ${CONFIG} 'platform.aws.region')
+aws_region=$(/tmp/yq r "${CONFIG}" 'platform.aws.region')
```

Comment:
https://github.com/openshift/release/pull/10105/files#diff-27605b7c8f643fa9b7d430f76604ebe6R10 is already quotes CONFIG, why do we need to quote again ??

this shellcheck warnings is just _too noisy_ imo

Reply:
The earlier quotes affect word splitting on that line.  Tgese quotes affect splitting on this line.  For example:

```console
$ A='b c'
$ B="${A}"
$ python -c 'import sys; print(sys.argv)' ${B}
['-c', 'b', 'c']
$ python -c 'import sys; print(sys.argv)' "${B}"
['-c', 'b c']
```